ctf/tests/core_suite.lua

```diff
@@ -0,0 +1,91 @@
+local lunatest = require("lunatest")
```

Comment:
I confirm asl97 here.
"core_suite.lua" is a file only to test "core.lua".
I'll make "teams_suite.lua" to test "teams.lua" or even "teams_join_suite.lua" to test specific part of "teams.lua".